src/marginaldamage.jl

```diff
@@ -10,12 +10,12 @@ TODO:
 - do we want to implement ramsey discounting, i.e. offer the eta keyword argument as well? 
 - should marginal damages be a step function across the ten year timesteps to be used by the SCC? or should I lineraly interpolate to get annual marginal emissions?
 """
```

Comment:
need to update the docstring to reflect the new `pulse_size` argument (and can we just delete these TODO's that are here as well?)